tests/test_basic.py

```diff
@@ -174,6 +180,8 @@ def test_projection(test_data, fused: bool, calc_compensations: bool):
     quats = test_data["quats"]
     scales = test_data["scales"]
     means = test_data["means"]
+    ortho = test_data.get("ortho", False)
```

Comment:
You can set `ortho` as an argument of this function and decorate this function with `pytest` to specify this argument to be with options True and False. This way pytest would test over all choices. See examples here: https://github.com/nerfstudio-project/gsplat/blob/45d196a3611e627c498f237c27014cdaf42d94d2/tests/test_rasterization.py#L21-L23